lib/gds_api/content_api.rb

```diff
@@ -52,13 +52,12 @@ def child_tags(tag_type, parent_tag, options={})
   end
 
   def tag(tag, tag_type=nil)
-    url = "#{base_url}/tags"
-
-    if tag_type
-      url << "/#{CGI.escape(tag_type)}"
+    if tag_type.nil?
```

Comment:
In which case it may be worth defaulting to section, but outputting a deprecation warning (eg https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/router.rb#L48-L51)